core/__init__.py

```diff
@@ -35,11 +36,61 @@ def save():
         # filef.write(block_chain._to_json().encode("utf-8"))
         return "blockchain saved"
 
+    @app.route("/user", methods=["POST", "GET"])
+    def user():
+        curr_user = User()
+        if request.method == "POST":
+            if request.form["submit_button"] == "add_transaction":
+                recipient = request.args.get("recipient")
+                amount = request.args.get("amount")
+                coin = request.args.get("coin")
+                transaction = curr_user.generate_transaction(recipient, coin, amount)
+                block_chain.add_new_transaction(transaction)
+                return curr_user.private_key.exportKey("PEM")
+            if request.form["submit_button"] == "set_user":
+                privatekey = request.args.get("privatekey")
+                curr_user.set_keys(privatekey)
+        # filef = open(document_path, "wb")
+
+        # filef.write(block_chain._to_json().encode("utf-8"))
```

Comment:
Can we delete out the commented code or is there a plan for it.

Reply:
deleted